content/zh-cn/docs/reference/config-api/apiserver-encryption.v1.md

```diff
@@ -24,9 +28,59 @@ auto_generated: true
 
 ## `EncryptionConfiguration`     {#apiserver-config-k8s-io-v1-EncryptionConfiguration}
 
-<p><!--EncryptionConfiguration stores the complete configuration for encryption providers.-->
+<!--
+EncryptionConfiguration stores the complete configuration for encryption providers.
+It also allows the use of wildcards to specify the resources that should be encrypted.
+Use '<em>.' to encrypt all resources within a group or '</em>.<em>' to encrypt all resources.
+'</em>.' can be used to encrypt all resource in the core group.  '<em>.</em>' will encrypt all
+resources, even custom resources that are added after API server start.
+Use of wildcards that overlap within the same resource list or across multiple
+entries are not allowed since part of the configuration would be ineffective.
+Resource lists are processed in order, with earlier lists taking precedence.
+-->
+<p>
 EncryptionConfiguration 为加密驱动保存完整的配置信息。
+它还允许使用通配符指定应加密的资源。
+使用 '<em>.<!-- raw HTML omitted --></em>' 加密组内的所有资源，或使用 '<em>.</em>' 加密所有资源。
```

Comment:
The upstream English content is incorrect. I'm fixing it in another PR.
Please hold.
/hold

Reply:
Already updated with the lasted en text in #41035